Cargo.lock

```diff
@@ -1813,7 +1813,7 @@ name = "rand_chacha"
 version = "0.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "rand_core 0.3.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "rand_core 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Why would upgrading `dlmalloc` downgrade `rand_core`?

Reply:
Both of those `rand_core` versions are already in the dependency tree, and `rand_chacha` requires `rand_core >= 0.2, < 0.4`. I guess it's just a quirk of how cargo resolved this when making the other change. But `rand_core` 0.2.2 just re-exports from 0.3 anyway, so it doesn't really matter.